source/_patterns/00-atoms/references/reference.mustache

```diff
@@ -39,11 +41,23 @@
     {{> atoms-doi}}
   {{/doi}}
 
+  {{^doi}}
```

Comment:
this has snuck in from https://github.com/elifesciences/pattern-library/pull/1063, I think

Reply:
I had to put this in so I could style this up. Can be removed when we merge in to master. If there is another reason can you let me know.